benefits/static/css/styles.css

```diff
@@ -143,61 +143,115 @@ h3,
   font-size: var(--h3-font-size);
 }
 
-/* making the sticky footer */
-html,
-body {
-  height: 100%;
-  overflow-y: unset;
-}
-
-body {
-  display: flex;
-  flex-direction: column;
-}
-
+/* Main */
+/* The minimum height is calculated by 100 viewport height minus Header and Footer height */
 main {
-  flex-shrink: 0 !important;
+  min-height: var(--main-content-min-height);
 }
 
-main#main-content.main-content {
-  position: relative;
-}
+/* Footer */
+/* Footer has same font styles on all screen widths */
+/* Mobile first: One link per row, each link is 50px height */
+/* Screen width above 992px - Footer is 50px height */
+/* Note: Not all Footer styles are written in mobile-first */
 
-/* All pages but the Start Page */
-main .main-row {
-  min-height: calc(100vh - 120px);
+:root {
+  --footer-background-color: var(--dark-color);
+  --footer-link-color: #73b3e7;
+  --footer-link-width: 100%;
+  --footer-link-hover-color: #9b74d7;
+  --footer-link-font-weight: 500;
+  --footer-mobile-underline-color: var(--bs-white);
 }
 
-main .main-row .col-lg-6.image {
-  background: 48% 75% url("/static/img/ridertappingbankcard.png") no-repeat;
-  background-size: cover;
+@media (min-width: 767px) {
+  :root {
+    --footer-link-width: auto;
+  }
 }
 
 footer {
-  margin-top: auto;
-  padding-top: 1rem;
-  padding-bottom: 1rem;
+  background: var(--footer-background-color);
+}
+
+footer .footer-links li {
+  width: var(--footer-link-width);
 }
 
-footer.global-footer .footer-links a {
+footer .footer-links li a {
   color: var(--footer-link-color);
-  font-weight: var(--bs-body-font-weight);
+  font-weight: var(--footer-link-font-weight);
+  font-size: var(--bs-body-font-size);
   text-decoration: underline;
+  letter-spacing: 0;
+  line-height: 50px;
 }
 
-footer.global-footer .footer-links a:hover,
-footer.global-footer .footer-links a:focus,
-footer.global-footer .footer-links a:active {
+footer .footer-links li a:hover,
+footer .footer-links li a:focus,
+footer .footer-links li a:active,
+footer .footer-links li a:visited {
   color: var(--footer-link-hover-color);
 }
 
-/* class styles */
+@media (max-width: 992px) {
```

Comment:
Can you please call out here with a comment that this is a non-mobile-first media query? And maybe elsewhere too if needed... will help us go back and clean up later.

Reply:
I will add a comment into the code and also put it here: This is because of the way the Footer on Mobile _must_ have 100% width fully to get the full-width white line. And the Footer on Desktop+ _needs_ the Bootstrap `.container` logic to size its widths. If we removed the `.container` and went with `width: 100%` (to be mobile-first), then we'd have to re-write our own container width logic, which would be very difficult. So basically, it's because of the full-width line that goes edge to edge.

Reply:
Makes sense and that is totally fine! A little doc comment describing that would be helpful is all.